### ISSUE TITLE
Use only primary hash to increase performance

### ISSUE THREAD

**Issue by Byron** [Byron/godi#23]
Turns out that MD5 is the fastest of the two hashes (MD5, Sha1) currently used, doing 600MB/s, whereas Sha1 'only' does 450MB/s. CRC64 clocks in at the same speed, whereas CRC32 only does 310MB/s.
In conclusion, MD5 is a perfectly valid backup hash, and not taking it wouldn't make anything faster in our case as we do it in parallel anyway.

It shows that simply not taking the MD5 as backup hash has a huge performance impact, bumping speed from 1.2GiB/s to 1.7GiB/s. Sha1 is the better hash of the two, maybe something like this could do it:

``` bash
godi --no-backup-hash seal A/
# Detects that MD5 is unset and doesn't do MD5 itself
godi verify A/godi.gobz
# Is forced to not do MD5
godi --no-backup-hash verify A/godi.gobz
```

It's clearly a nice to have, but saving CPU time saves energy, which is why it's a valid request.
